app/views/home/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1 class="text-center">Khan Academy Myanmar Community</h1>
-<p>This is the home page.</p>
+<div class="row">
+	<div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12" style="float:left;text-align:center;margin-top:100px;">
```

Comment:
On a side note, we should figure out a consistent way to add vertical whitespace. I've been using empty divs, but you seem to prefer hardcoding padding into the element itself. I don't think either way is terrible, but we should stick with one.